Dockerfile

```diff
@@ -11,7 +11,7 @@ LABEL maintainer OSG Software <help@opensciencegrid.org>
 RUN groupadd -o -g 10941 squid && \
     useradd -o -u 10941 -g 10941 -s /sbin/nologin -d /var/lib/squid squid && \
     yum update -y && \
-    yum install -y frontier-squid --enablerepo=osg-testing && \
+    yum install -y frontier-squid && \
```

Comment:
do you not want to `--enablerepo` based on `YUM_BASE_REPO` or something?

Reply:
We get this from the selection of the `software-base` image above https://github.com/opensciencegrid/docker-software-base/blob/master/Dockerfile#L32. In similar PRs I've added upcoming repo enabling based on the `BASE_YUM_REPO` and maybe we should just enable the relevant upcoming repo in `software-base`

Reply:
Oh neat, i didn't realize (eg) the testing image permanently enabled the osg-testing yum repo